app/Site/Models/User.php

```diff
@@ -151,6 +151,10 @@ class User extends Authenticatable implements CommunityMember, Developer, HasCom
         "ContribYield",
         "TrueRAPoints",
         "websitePrefs",
+        "Motto",
+        "Untracked",
+        "LastLogin",
+        "avatarUrl",
```

Comment:
I am not sure if it is good practice to expose these. This way the only way to have these natively available when calling `toArray()`. Another alternative would be to delete these lines and instead do something like:

```php
return array_merge(
    $foundUser->toArray(),
    [
        'avatarUrl' => $foundUser->AvatarUrl,
        // ...
    ]
);
```

Reply:
I'm under the impression that it's fine to expose anything that's not sensitive (email address, encrypted password, api token, etc).